apps/bare-expo/ios/Podfile.lock

```diff
@@ -159,9 +159,17 @@ PODS:
     - React
   - expo-dev-menu (0.4.1):
     - expo-dev-menu-interface
+    - expo-dev-menu/Main (= 0.4.1)
```

Comment:
I believe any change in this file that is not directly connected with the module you're working on right now should be extracted as a separate PR 😉 
```suggestion
```